### ISSUE TITLE
MP2.5 happily creates PGDB with zero genes, zero pathways

### ISSUE THREAD

**Issue by taltman** [hallamlab/metapathways2#19]
There was no indication via the GUI or the log-files that something went wrong. I'm not sure how to debug this to find out where the problem started. Suggestions?

The bug is that MP should do basic sanity-checks to make sure that stages succeeded or failed.

**Comment by nielshanson** [comment]
Hey Tomer,

Could you check one of your output directories for the content [sample_name]/ptools/ directory. This should give us an indication of the input to pathway tools to figure out an empty PGDB was created.

**Comment by taltman** [comment]
in my /ptools directory, I have a single PathoLogic formatted file, called '0.pf'. It is empty. Looks like a bug.

**Comment by taltman** [comment]
I have a theory: Looks like something is wrong with my "stock" MPv2.5 install, and all of the protein (fragments) were annotated only with "hypothetical protein". PathoLogic probably deleted all of those entries.

MP should "just work" out-of-the-box. If there's additional steps required then:
1. They should be documented, and 
2. When they are not obeyed, MP should catch them and fail early and noisily.

**Comment by taltman** [comment]
I think I figured out what went wrong.

I didn't notice at first that on the Parameters page, that you had to select the check-boxes next to the database names. I assumed that all of them would be used automatically. So I failed to select any databases, MP was happy to chug along anyways, and I was very confused with the output.

So there are two different issues here:

1. MP should always have sensible default values. Having no functional nor taxonomic databases selected is not sensible. I don't see anything wrong with having the bundled databases selected by default.

2. If the user modifies the default values to something which is not sensible, the user should be prevented from proceeding, or at least, a warning message should be brought to their attention.
